extensions/amp-auto-ads/0.1/ad-network-config.js

```diff
@@ -0,0 +1,94 @@
+/**
+ * Copyright 2016 The AMP HTML Authors. All Rights Reserved.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *      http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS-IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+import {DataAttributeDef} from './placement';
+import {buildUrl} from '../../../ads/google/a4a/url-builder';
+import {user} from '../../../src/log';
+
+/** @const */
+const TAG = 'amp-auto-ads';
+
+/**
+ * An interface intended to be implemented by any ad-networks wishing to support
+ * amp-auto-ads.
+ * @interface
+ */
+export class AdNetworkConfig {
```

Comment:
It seems weird for it not to be exported since it's the interface that's returned by the exported function getAdNetworkConfig.

Also, if I do remove export from it, then gulp lint complains because it thinks it's not used, since it only gets used by @implements statements in the jsDoc. Any way around this?